pom.xml

```diff
@@ -33,11 +33,28 @@
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-actuator</artifactId>
         </dependency>
+
         <dependency>
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
             <optional>true</optional>
         </dependency>
+
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-jpa</artifactId>
+        </dependency>
+
+        <dependency>
```

Comment:
@mcruzdev this is usually not good, as now our application depends on mysql, and if someone wants to configure it with another database provider they can't. What people usually do is to create a base docker image without any driver and then another layer which adds the jdbc driver of your choice.. can we do that here as well?

Reply:
How can i do that ?
I do not know to do .. but i will search about.